package-lock.json

```diff
@@ -4746,27 +4781,26 @@
       "license": "MIT"
     },
     "node_modules/@openapitools/openapi-generator-cli": {
-      "version": "2.20.2",
-      "resolved": "https://registry.npmjs.org/@openapitools/openapi-generator-cli/-/openapi-generator-cli-2.20.2.tgz",
-      "integrity": "sha512-dNFwQcQu6+rmEWSJj4KUx468+p6Co7nfpVgi5QEfVhzKj7wBytz9GEhCN2qmVgtg3ZX8H6nxbXI8cjh7hAxAqg==",
+      "version": "2.23.1",
```

Comment:
we have this package in our package.json
"@openapitools/openapi-generator-cli": "^2.20.2",

so it should be change there as well